src/style.css

```diff
@@ -1 +1,31 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+ul {
```

Comment:
<img width="1004" alt="image" src="https://github.com/user-attachments/assets/240dab27-c6e3-4057-b2b8-da93d9cc8130">

Reply:
I have a question in ul can I leave padding:0 and margin: 0;  (its like if i have ul in other places, and I just want to reset in general for every ul? Or should i do the reset in every class in ul?)